### PR TITLE
Fix reducer tests

### PR DESCRIPTION
**Pull request by mscharfstein**
Fix reducer tests

Works on #9

### CODE REVIEW

File: client/src/features/form/formSlice.js [hunk 1]
```diff
@@ -5,9 +5,7 @@ export const formSlice = createSlice({
   initialState: [],
   reducers: {
     formUpdate: (state, action) => {
-      action.payload.form.map((formSection) => {
-        state.push(formSection);
-      });
+      action.payload.form.map((formSection) => state.push(formSection));
```
Comment (marcja) [inline client/src/features/form/formSlice.js:8]: As discussed, this is a bug. The function passed into `map` requires a return value and state.push doesn't return anything. Maybe you meant `Array.forEach` or `[formSection, ...state]`?

File: client/src/features/form/formSlice.test.js [hunk 2]
```diff
@@ -1,23 +1,21 @@
-import "@testing-library/jest-dom/extend-expect";
-import form, { formUpdate } from "./formSlice";
+import '@testing-library/jest-dom/extend-expect';
+import formReducer, {formUpdate} from './formSlice';
```
Comment (marcja) [inline client/src/features/form/formSlice.test.js:2]: As long as you're renaming... s/formReducer/reduceForm/g?

File: client/src/features/form/formSlice.test.js [hunk 3]
```diff
@@ -1,23 +1,21 @@
...
   ];
   expect(
-    form([], {
-      type: formUpdate.type,
-      payload: {
-        form: form,
-      },
-    })
+      formReducer([], {
+        type: formUpdate.type,
```
Comment (marcja) [inline client/src/features/form/formSlice.test.js:15]: s/formUpdate/updateForm/g?

File: client/src/features/questions/questionsSlice.test.js [hunk 4]
```diff
@@ -1,34 +1,28 @@
 import '@testing-library/jest-dom/extend-expect';
-import questions, {questionsUpdate} from './questionsSlice';
+import questionsReducer, {questionsUpdate} from './questionsSlice';
```
Comment (marcja) [inline client/src/features/questions/questionsSlice.test.js:2]: s/questionsReducer/reduceQuestions/g?

File: client/src/features/form/formSlice.js (hunk 1 above)
Comment (mscharfstein) [reply client/src/features/form/formSlice.js]: forEach works

**Pull request merged**